### ISSUE TITLE
Another import issue 

### ISSUE THREAD

**Issue by chromafunk** [google/google-api-python-client#168]
I have tried to test the api on Python3 and I have problems : 

ImportError: No module named apiclient.discovery

I have installed it via pip3 and I I have httplib2 and urltemplate, why is this happening ? 

I have created a virtualenv with Python 2.7 and it also throws the same error ...

**Comment by chromafunk** [comment]
Ok, I have fixed this partly by reinstalling dependencies and google-api-client itself ... but one of the code examples for youtube upload uses both httplib2 and httplib, but I can't install httplib. Also on the python documentation they say they have renamed httplib to http.client : Note The httplib module has been renamed to http.client in Python 3. The 2to3 tool will automatically adapt imports when converting your sources to Python 3.

So I guess the examples are only for python 2.7, correct ?

**Comment by nathanielmanistaatgoogle** [comment]
What's the text of the example that you're following?

If it says `import httplib` then yes, it's a 2.7-only example.

**Comment by chromafunk** [comment]
@nathanielmanistaatgoogle  I've fixed this a long time ago, now I have working code adapted to Python3, the only problem I am still facing is when using resumable = False, I have commented that on the patch commit for that bug. Again, if you want me to dump the error it's easy to reproduce for me, let me know if you want it. I can't help further on fixing the bug because I have not enough experience to do that yet.
